components/rowListScene.brs

```diff
@@ -1,12 +1,21 @@
 sub init()
     m.global.addField("http", "node", FALSE)
     m.global.addField("ratio", "float", FALSE)
+    m.global.addField("mainScreen", "node", FALSE)
+    m.global.mainScreen = m.top
     m.global.http = createObject("roSGNode", "httpTask")
     m.global.http.control = "RUN"
     m.global.ratio = 1
     m.rowList = m.top.findNode("mainRowList")
+    m.MainBoldLabel = m.top.findNode("MainBoldLabel")
+    m.DateAndDurationLabel = m.top.findNode("DateAndDurationLabel")
+    m.DescriptionLabel = m.top.findNode("DescriptionLabel")
+    m.CategoryLabel = m.top.findNode("CategoryLabel")
+    m.textAnimation = m.top.findNode("heroDetailsAnimation")
```

Comment:
It is better (visually) to have a fade-out animation and then a fade-in one.

Reply:
I don't see a fade-out animation for the old background. Also `m.overlayAnimation` is never used.

Reply:
Done

Reply:
It is still not correct. Please don't resolve issues yourself. I should do that to track things.

Reply:
Alright